test/db/formats/coff

```diff
@@ -105,22 +93,17 @@ EXPECT=<<EOF
  8 fd: 5 +0x00000eec 0x00000d60 - 0x00000e33 r-- vmap..debug$S
  9 fd: 5 +0x00000ff2 0x00000e40 - 0x00000e43 r-- vmap..rtc$IMZ
 10 fd: 5 +0x00001000 0x00000e50 - 0x00000e53 r-- vmap..rtc$TMZ
-[Sections]
-
-nth paddr        size vaddr       vsize perm name       
---------------------------------------------------------
-0   0x0000017c   0xef 0x00000000   0xef ---- .drectve
-1   0x0000026b  0xab0 0x000000f0  0xab0 -r-- .debug_S
-2   0x00000d1b   0x74 0x00000ba0   0x74 -r-- .debug_T
-3   0x00000d8f   0x2d 0x00000c20   0x2d -r-x .text_mn
-4   0x00000dbc   0xd4 0x00000c50   0xd4 -r-- .debug_S_1
-5   0x00000ec2   0x2a 0x00000d30   0x2a -r-x .text_mn_1
-6   0x00000eec   0xd4 0x00000d60   0xd4 -r-- .debug_S_2
-7   0x00000ff2    0x4 0x00000e40    0x4 -r-- .rtc_IMZ
-8   0x00001000    0x4 0x00000e50    0x4 -r-- .rtc_TMZ
-
-[Symbols]
-
+paddr      size  vaddr      vsize align perm name       type flags 
+-------------------------------------------------------------------
+0x0000017c 0xef  0x00000000 0xef  0x0   ---- .drectve        
```

Comment:
I am not sure removing the number of section is helpful. Referring to some section by the number is useful for the text description of the Rizin output.

Reply:
In which case is it useful? I removed those numbers because they weren't coming from the binary, but they were just some indexes incremented by rizin. What do you mean by "text description of the rizin output"?

Reply:
@ret2libc if you put Rizin output in the article or some kind of report. I don't have *strong opinion* about that, just think it is better with numbers.